packages/react-magma-dom/src/components/StyledButton/index.tsx

```diff
@@ -19,17 +19,15 @@ import { ThemeContext } from '../../theme/ThemeContext';
 import { ButtonProps } from '../Button';
 
 interface StyledButtonProps extends ButtonProps {
-  as?: any;
   href?: string;
   iconOnly?: boolean;
-  to?: string;
 }
 
 export const buttonStyles = props => css`
   align-items: center;
   background: ${buildButtonBackground(props)};
   border: ${props.variant === 'outline' ||
-  (props.variant === 'solid' && props.color === 'secondary' && !props.isInverse)
+  (props.variant !== 'link' && props.color === 'secondary' && !props.isInverse)
```

Comment:
These should be updated in a future PR to use the enum values in the comparisons.